docs/build-on-linea/tooling/node-providers.md

```diff
@@ -12,7 +12,8 @@ title: Node Providers
 - [GetBlock](https://getblock.io/)
 - [ANKR](https://www.ankr.com/rpc/)
 
-If you want your own private personal RPC endpoint, you can [run your own Linea node](/docs/build-on-linea/run-a-node.mdx).
+If you want your own private personal RPC endpoint, you can [run your own Linea node](/docs/build-on-linea/run-a-node.mdx) 
```

Comment:
Hmm, I think we want the Linea docs to remain credibly neutral, so maybe can we change to something like

```
## Run your own node

- [Set it up yourself](/docs/build-on-linea/run-a-node.mdx)
- [One-click deploy with EasyNode](https://app.easy-node.xyz/)
```

So that it doesn't seem as much like an explicity recommendation?

Reply:
I made the recommended changes